review04/passenger-service/src/main/java/cz/muni/fi/airportmanager/passengerservice/entity/Passenger.java

```diff
@@ -0,0 +1,120 @@
+package cz.muni.fi.airportmanager.passengerservice.entity;
+
+import cz.muni.fi.airportmanager.passengerservice.model.CreatePassengerDto;
+import jakarta.persistence.CascadeType;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@Entity
+// TODO Add entity annotation
+// TODO Add id field with generated value and it is a primary key
+// TODO Add getters and setters
+// TODO add relation to Notification entity (you will need to change the Notification entity as well)
+public class Passenger {
+    @Id
+    @GeneratedValue
+    private Long id;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private Long flightId;
+
+    @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER)
```

Comment:
This is not having owning side? Where is ManyToOne in notification?

Reply:
I've tried to resolve this by using every possible combination. Unfortunately, when I added ManyToOne to the other side, I didn't fetch it and wasn't able to understand why. I'll probably create an issue for this.